.travis.yml

```diff
@@ -2,11 +2,11 @@ language: erlang
 sudo: false
 
 otp_release:
-- 18.1
+- 18.2
 
 before_install:
-- wget http://s3.hex.pm/builds/elixir/v1.1.1.zip
-- unzip -d elixir v1.1.1.zip
+- wget http://s3.hex.pm/builds/elixir/v1.2.1.zip
```

Comment:
Should we not get the latest release (1.2.4) while we're at it, since we're gonna update everything?